src/Components/Navigation.jsx

```diff
@@ -4,6 +4,21 @@ const Navigation = () => {
     return (
         <nav>
             <Link to="/api/reviews">All Reviews</Link>
+            <span> | </span>
+            <Link to="/api/reviews?category=deck-building">Deck Building</Link>
+            <span> | </span>
+            <Link to="/api/reviews?category=dexterity">Dexterity</Link>
+            <span> | </span>
+            <Link to="/api/reviews?category=engine-building">Engine Building</Link>
+            <span> | </span>
+            <Link to="/api/reviews?category=hidden-roles">Hidden Roles</Link>
+            <span> | </span>
+            <Link to="/api/reviews?category=push-your-luck">Push Your Luck</Link>
+            <span> | </span>
+            <Link to="/api/reviews?category=roll-and-write">Roll and Write</Link>
+            <span> | </span>
+            <Link to="/api/reviews?category=strategy">Strategy</Link>
+
```

Comment:
If you wanted to neaten this up, you could get all categories from the api and map over the list, making a link for each! Upto you though